components/automate-ui/src/styles/_variables.scss

```diff
@@ -42,6 +42,9 @@ $default-logo: url('/assets/img/logos/AutomateLogo-default.svg');
 $preview-logo: url('/assets/img/logos/AutomateLogo-preview.svg');
 $beta-logo: url('/assets/img/logos/AutomateLogo-beta.svg');
 
+// desktop
+$chef-desktop-light: $lavender-light-2;
```

Comment:
named one color - but decided should hold off on more until we figure out naming convention for all these new colors.